### DISCUSSION TITLE
in between two videos

### DISCUSSION THREAD

**Discussion by p0mad** [B34STW4RS/AD-Evo-Tuner#8] [General]
I have two videos and i want to interpolate between those!
can i use you repo? with possible solution
Thanks

**Comment by B34STW4RS** [comment]
What you could do is take a repo that is good for inference(this one isn't) and has init image, and modify it to take both an init start image and an init end image, and set them as the latents. So you would then take the end image from the end of one clip, and a start image from the beginning of the second clip and interpolate them with AD. Something like that.

**Reply by p0mad** [reply]
@B34STW4RS 
Thanks for the reply.
What do you mean by:
> take a repo that is good for inference(this one isn't)

Can you please name a few good repos for this?

if i get another repo, How can i 
> interpolate them with AD ?

Could you be more specific on how to give two init imaegs(last_old,first_new) to the code as latents?

Thanks
Best regards

**Reply by B34STW4RS** [reply]
You can try visioncrafter, AnimateDiff-cli, or you can use the Comfy addons, there are some in our discord. Please feel free to join the discussion there. https://discord.gg/mzAtxzMS
